### DISCUSSION TITLE
Is there any way to use OBS api to get screenshots in my own script?

### DISCUSSION THREAD

**Discussion by mrr00b00t** [obsproject/obs-studio#5490] [General]
I am interest in use the bind mechanism and get screenshots from windows to my own script, is there any way to do this with some user friendly api?

**Comment by WizardCM** [comment]
Are you purely looking to save a screenshot to the filesystem, or are you looking to do some analysis/manipulation?

**Comment by Fenrirthviti** [comment]
Please use the Forums or Discord for support.

https://obsproject.com/help
